src/ripple/rpc/impl/GetAccountObjects.cpp

```diff
@@ -55,17 +55,17 @@ getAccountObjects (Ledger const& ledger, AccountID const& account,
             iter = std::find (iter, entries.end (), entryIndex);
             if (iter == entries.end ())
                 return false;
-            
+
             found = true;
         }
 
         for (; iter != entries.end (); ++iter)
         {
-            auto const sleNode = cachedRead(ledger, *iter );
+            auto const sleNode = ledger.read(keylet::unchecked(*iter));
```

Comment:
If we're iterating the children of a directory then the correct `Keylet` is `keylet::child(*iter)`